src/web/mjs/connectors/Dm5.mjs

```diff
@@ -0,0 +1,125 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Dm5 extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'dm5';
+        super.label = 'DM5 漫画';
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'https://www.dm5.com';
+        this.path = '/manhua-list-p%PAGE%/';
+
+        this.queryMangas = 'div.box-body ul li div.mh-item div.mh-item-detali h2.title a';
+        this.queryChapters = 'div#chapterlistload ul#detail-list-select-1 li a';
+        this.queryPages = 'div#showimage div#cp_img img#cp_image';
+        this.queryMangaTitle = 'div.banner_detail_form div.info p.title';
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        for(let page = 1, run = true; run; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangas.length ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        const uri = new URL(this.path.replace('%PAGE%', page), this.url);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryMangas);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.title.trim()
+            };
+        });
+    }
+
+    async _getChapters(manga) {
+        // https://regex101.com/r/vD6dIl/2/
+        const regex = /（(.*?)）/gim;
+        const script = `
+            new Promise((resolve, reject) => {
+                setTimeout(() => {
+                    try {
+                        let button = document.querySelector('#checkAdult');
+                        if(button) {
+                            button.click();
+                        }
+                        let chapterList = [...document.querySelectorAll('${this.queryChapters}')].map(element => {
+                            return {
+                                id: new URL(element.href, window.location).pathname,
+                                title: element.text.replace(${regex}, '').trim()
+                            };
+                        });
+                        resolve(chapterList);
+                    } catch(error) {
+                        reject(error);
+                    }
+                }, 500);
+            });
+        `;
+        const uri = new URL(manga.id, this.url);
+        const request = new Request(uri, this.requestOptions);
+        return await Engine.Request.fetchUI(request, script);
+    }
+
+    async _getPages(chapter) {
+        const script = `
```

Comment:
This could possibly be simplified, take a look how this is done in MangaFox.
From a brief view the best option is to use the mobile domain `m.dm5.com` and then just access the `newImgs` variable on the chapter page.